std_msgs/Header header
int64 deco_count
int64 box_num
sensor_msgs/Image back_img
---
geometry_msgs/Point[] real_deco_uv
